plugins/com.google.cloud.tools.eclipse.appengine.ui/plugin.properties

```diff
@@ -1,9 +1,14 @@
 pluginName=Cloud Tools for Eclipse User Interface
```

Comment:
We had that before and it sounded weird as it (now) contributes UI to the IDE.  How about `Cloud Tools for Eclipse App Engine UI Support`?